Sources/IBLinterKit/Config/Config.swift

```diff
@@ -74,6 +79,7 @@ public struct Config: Codable {
         viewAsDeviceRule = try container.decodeIfPresent(Optional<ViewAsDeviceConfig>.self, forKey: .viewAsDeviceRule) ?? nil
         reporter = try container.decodeIfPresent(String.self, forKey: .reporter) ?? "xcode"
         disableWhileBuildingForIB = try container.decodeIfPresent(Bool.self, forKey: .disableWhileBuildingForIB) ?? true
+        ignoreCache = try container.decodeIfPresent(Bool.self, forKey: .ignoreCache) ?? false
```

Comment:
What do you think about default value? Should we enable cache by default?

Reply:
Yes enabled by default is the good choice if there is no way to forgot some issues